environment.yml

```diff
@@ -3,12 +3,13 @@ channels:
 dependencies:
 - python >= 3.9
 - click
+- fiona
 - geopandas-base
 - numpy
 - pandas
 - pip
+- requests
 - shapely
 - tqdm
 - utm
-- pip:
-  - unzip-http
```

Comment:
oh could we actually leave this as a pip install for the moment until/if we just remove `unzip-http`? 
I forgot why I added this, but for some weird reason [the conda package isn't python only](https://anaconda.org/conda-forge/unzip-http/files), so i can't conda install on my M1 Mac

Reply:
sure thing! thanks for updating the docs on this. worked great on intel mac.